graylog2-web-interface/src/stores/streams/StreamRulesStore.ts

```diff
@@ -75,6 +76,9 @@ class StreamRulesStore {
   _emitChange() {
     this.callbacks.forEach((callback) => callback());
   }
+  unregister(callback: Callback) {
+    lodash.pull(this.callbacks, callback);
```

Comment:
I know we use lodash in one or two places already, but I honestly see no benefit extending its use for things like this. IMHO using `filter` from the standard library is simple enough, does not require anybody to learn anything else, and does not require the external dependency.